### ISSUE TITLE
Unable to switch to card layout for Contacts

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#12587]
### Mautic Version

5.0.x series

### PHP version

8

### What browsers are you seeing the problem on?

Not relevant

### What happened?

You should be able to switch to card / grid layout using the keyboard shortcut 'c', but this is broken since the Twig refactor.

### How can we reproduce this issue?

Step 1: Go to contacts
Step 2: Press 'c'
Step 3: Note flash error message and log entry
Step 4: Note that you can't access contacts any more! Tip: Delete the session cookie and log in again to reset to table view!

### Relevant log output

```shell
[2023-07-26 14:56:56] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "@MauticLead/Lead/index.html.twig" (looked into: /var/www/html/app/bundles/LeadBundle/Resources/views)." at /var/www/html/app/bundles/LeadBundle/Resources/views/Lead/grid.html.twig line 19 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"@MauticLead/Lead/index.html.twig\" (looked into: /var/www/html/app/bundles/LeadBundle/Resources/views). at /var/www/html/app/bundles/LeadBundle/Resources/views/Lead/grid.html.twig:19)"} {"hostname":"mautic-web","pid":2081}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct
